tests/unittests/algo/test_configspace.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from orion.algo.space import Categorical, Fidelity, Integer, Real, Space, to_orionspace
+from orion.algo.space.configspace import IMPORT_ERROR, to_configspace
+
+
+@pytest.mark.skipif(IMPORT_ERROR, reason="Running without ConfigSpace")
+def test_orion_configspace():
+    space = Space()
+
+    space.register(Integer("r1i", "reciprocal", 1, 6))
+    space.register(Integer("u1i", "uniform", -3, 6))
+    space.register(Integer("u2i", "uniform", -3, 6))
+    space.register(Integer("u3i", "uniform", -3, 6, default_value=2))
+
+    space.register(Real("r1f", "reciprocal", 1, 6))
+    space.register(Real("u1f", "uniform", -3, 6))
+    space.register(Real("u2f", "uniform", -3, 6))
+    space.register(Real("name.u2f", "uniform", -3, 6))
+
+    space.register(Categorical("c1", ("asdfa", 2)))
+    space.register(Categorical("c2", dict(a=0.2, b=0.8)))
+    space.register(Fidelity("f1", 1, 9, 3))
+
+    space.register(Real("n1", "norm", 0.9, 0.1, precision=6))
+    space.register(Real("n2", "norm", 0.9, 0.1, precision=None))
+    space.register(Real("n3", "norm", 0.9, 0.1))
```

Comment:
which non supported priors ?

Reply:
`Fidelity` is there

Reply:
The behaviour will be like fidelity, I added the alpha prior for completeness,
& also added a test for the unsupported priors from configspace

Reply:
Should we make the conversion just ignore unsupported priors? For the fidelity we know it makes sense to drop it because the algorithms are expected to handle it differently. For the other priors I would be worried that the code crash at some latter weird places, like during wrapper's suggest() trial transformation. That would be hard to understand for the user compared to a crash during space conversion.

Reply:
updated